source/gui/settingsDialogs.py

```diff
@@ -1636,31 +1636,53 @@ def makeSettings(self, settingsSizer):
 		settingsSizer.Add(sizer)
 
 		# Translators: The label for the edit field in symbol pronunciation dialog to change the pronunciation of a symbol.

```

Comment:
I realise you didn't write this translators comment, but while you're here, can you please change "edit field" to "group of controls"?